src/main/groovy/io/github/cosmicsilence/utils/Gradle.groovy

```diff
@@ -0,0 +1,7 @@
+package io.github.cosmicsilence.utils
+
+class Gradle {
+    static boolean isVersion4(project) {
+        project.gradle.gradleVersion.startsWith("4")
```

Comment:
`.startsWith("4.")`?

Reply:
Changed and moved to GradleCompat class.